test/lib/versioncheck-test.js

```diff
@@ -9,20 +9,20 @@ var apiHost = 'https://mycommunity.com';
 var testRoot = path.resolve(__dirname) + '/..';
 var spawn = require('child_process').spawn;
 var rewire = require('rewire');
+var serverMocks = require('./server-mocks');
 var badErrorRespnse = 'Bad response';
-var successVersion = 15.7;
-var higherVersion = '16.1';
-var higherVersion2 = '15.7.1';
-var higherVersion3 = '15.10';
+var successVersion = 16.2;
+var higherVersion = '16.4';
+var higherVersion2 = '16.3.1';
+var higherVersion3 = '16.3';
```

Comment:
Can we change this test to 16.10? The idea was to test for > 1 digit minor version.